Common/cpp/ReanimatedRuntime/ReanimatedRuntime.cpp

```diff
@@ -0,0 +1,45 @@
+#include "ReanimatedRuntime.h"
+
+#include <cxxreact/MessageQueueThread.h>
+#include <jsi/jsi.h>
+
+#include <memory>
+#include <utility>
+
+#if JS_RUNTIME_HERMES
+#include "ReanimatedHermesRuntime.h"
+#elif JS_RUNTIME_V8
+#include <v8runtime/V8RuntimeFactory.h>
+#else
+#include <jsi/JSCRuntime.h>
+#endif
+
+namespace reanimated {
+
+using namespace facebook;
+using namespace react;
+
+std::shared_ptr<jsi::Runtime> ReanimatedRuntime::make(
+    std::shared_ptr<MessageQueueThread> jsQueue) {
+#if JS_RUNTIME_HERMES
+  std::unique_ptr<facebook::hermes::HermesRuntime> runtime =
+      facebook::hermes::makeHermesRuntime();
+  facebook::hermes::HermesRuntime &hermesRuntime = *runtime;
+
+  return std::make_shared<ReanimatedHermesRuntime>(
+      std::move(runtime), hermesRuntime, jsQueue);
+#elif JS_RUNTIME_V8
+  jsQueue->quitSynchronous();
+
+  auto config = std::make_unique<rnv8::V8RuntimeConfig>();
+  config->enableInspector = false;
```

Comment:
fwiw, if you want to enable debugging on v8, you could also set this flag as true.